docker/test/integration/features/python.feature

```diff
@@ -143,3 +143,17 @@ Feature: MiNiFi can use python processors in its flows
     When all instances start up
 
     Then one flowfile with the contents "Check successful!" is placed in the monitored directory in less than 30 seconds
+
+  @USE_NIFI_PYTHON_PROCESSORS
+  Scenario: A MiNiFi instance can update attributes through native python processor
+    Given a GenerateFlowFile processor with the "File Size" property set to "0B"
+    And a FailureWithAttributes processor
```

Comment:
We could put an UpdateAttribute before the FailureWithAttributes, to verify that the attributes are merged, not replaced. Maybe also set an old value for `error.message`, to verify/document that the existing attribute gets overwritten.

Reply:
Good point, actually there was a bug in there because in NiFi the python API uses `addAttribute` and we used the same, but in NiFi it updates the attribute if it exists, but MiNiFi C++ doesn't, we have to use `setAttribute` instead. Updated in 30268333d5fff6c48822cb54925b8189ff6d0e49